tests/import_test.go

```diff
@@ -184,6 +186,110 @@ var _ = Describe("[rfe_id:1115][crit:high][vendor:cnv-qe@redhat.com][level:compo
 
 })
 
+var _ = Describe("DataVolume Garbage Collection", func() {
```

Comment:
General questions about functional tests.
How aren't most of the tests failing after we started GCing? I have assumed we will need adjustments for this, but it doesn't look like it?

Reply:
It's currently disabled by default, and we set it only for those new tests.
However as noted, we will adapt all DV related tests (quite straightforward) and add a CI lane with GC on.

Reply:
Right, since for other tests the TTL is not set in the CDIConfig, correct?